src/main/java/core/basesyntax/service/impl/CsvDataWriterImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.DataWriter;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvDataWriterImpl implements DataWriter {
```

Comment:
This class should receive the report and write it to a file

Reply:
fixed